openstack/tools/charmed_openstack_functest_runner.sh

```diff
@@ -36,12 +39,22 @@ OPTIONS:
     --no-wait
         By default we wait before destroying the model after a test run. This
         flag can used to override that behaviour.
+    --manual-functests
+        Runs functest commands separately (deploy,configure,test)instead of
+        the entire suite.
+    --remote-build
+        Builds the charm in a remote location and transfers the charm file over.
+        The destination needs to be prepared for the build and authorized for
+        ssh. Implies --skip-build. Specify parameter as <destination>,<path>.
+        Example: --remote-build ubuntu@10.171.168.1,~/git/charm-nova-compute
```

Comment:
do we really need this? cant you do the build on vm then rsync it all to a container and run the tests?

Reply:
otherwise we are just adding more and more complexity when there are alternatives

Reply:
yes, we need this if we are going to script things to avoid human mistakes as much as possible and have everything performed by the script. If things are done manually someone could build the wrong version, rsync the wrong file, etc, and the resulting script that does not include those steps wouldn't capture the mistake.